tensorflow/python/keras/engine/base_layer.py

```diff
@@ -405,7 +405,11 @@ def build(self, input_shape):
 
   @doc_controls.for_subclass_implementers
   def call(self, inputs, **kwargs):  # pylint: disable=unused-argument
-    """This is where the layer's logic lives.
+    """This is where the layer's logic lives. The second parameter `kwargs`
+    is not used anywhere as of now. Note here that `call()` method in
+    `tf.keras` is little bit different from `keras` API. In `keras` API,
+    you can pass support masking for layers as additional arguements.
+    Whereas `tf.keras` has `compute_mask()` method to support masking.
```

Comment:
Let keep first line as it is. The [PEP rules for docstrings](https://www.python.org/dev/peps/pep-0257/) say that first line should be a short phrase, followed by an empty line and then a longer description.

Then, let's move the discussion about `kwargs` not being used to the documentation for `kwargs`

Then, the notes here can be a different paragraph, just above documenting the arguments

Reply:
done